### DISCUSSION TITLE
Let us hide external repositories from main page

### DISCUSSION THREAD

**Discussion by goteusz-maszyk** [community/community#123928] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

So once you report A SINGLE ISSUE into any repository it starts appearing in your list. It's quite annoying, as many of those repos are no longer relevant to me and I still see them on my sidebar. Please, make it so only if you watch/have write permissions you see them.

**Comment by ghost** [comment]
When did this start? So does mine! !

**Comment by ghost** [comment]
![3865F28BAFE3F84D081D527762CC0B60](https://github.com/community/community/assets/163424176/44399623-d0c2-441b-9efe-4fcedd2d2388)
![9EB1C7B1E870B69D94491036A48FB301](https://github.com/community/community/assets/163424176/3b27bedc-cf78-469d-8841-aa45cf221a94)
damn it! How to delete someone else's project? ? ? ? ?
